src/lib/formats/vgi_dsk.cpp

```diff
@@ -0,0 +1,144 @@
+// license:BSD-3-Clause
+// copyright-holders:Eric Anderson
+/*********************************************************************
+
+Micropolis VGI disk image format
+
+The format is essentially an HCS-ordered sector image, but with raw
+275 byte sectors. The sector format is: SYNC (0xFF), track, sector,
+10 byte "user data", 256 byte data, checksum, 4 byte ECC, ECC present
+flag.
+
+http://www.bitsavers.org/pdf/micropolis/metafloppy/1084-01_1040_1050_Users_Manual_Apr79.pdf
+
+*********************************************************************/
+
+#include <cstring>
+
```

Comment:
Please put standard library headers after project headers.  But are you actually using `<cstring>` at all?  I didn’t see any uses of it in a quick read of the file.  I saw use of `<vector>` and `<cstdint>` which MAME relies on pretty heavily.